pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -18,16 +21,11 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ python makeWrapper zip pandoc ];
 
-  patchPhase = "rm youtube-dl";
-
-  configurePhase = ''
-    makeFlagsArray=( PREFIX=$out SYSCONFDIR=$out/etc PYTHON=${python}/bin/python )
```

Comment:
There is no generated binary, and I don't believe that it does. If that is something that people use it could be added to the makeFlags array, not the configurePhase.